api_tests/subjects/views.py/test_subject_list.py

```diff
@@ -0,0 +1,111 @@
+import pytest
```

Comment:
Tiny thing: did you mean to name this test directory `views.py/`?

Reply:
gah. thanks